Source/GameBaseFramework/Private/Components/GBFUIDialogManagerComponent.cpp

```diff
@@ -369,3 +365,8 @@ void UGBFUIDialogManagerComponent::RemoveAllDialogsFromViewport()
 
     ZOrder = 2;
 }
+
+TWeakObjectPtr< APlayerController > UGBFUIDialogManagerComponent::GetPlayerController() const
+{
+    return Cast< APlayerController >( GetOwner() );
+}

```

Comment:
Function can be inlined

Reply:
But that function should return a naked pointer, not a TWeakObjectPtr